frontend-react/src/styles/_reportstream.scss

```diff
@@ -261,6 +261,15 @@ ul.rb li .timestamp {
     padding-bottom: 0.5em;
 }
 
+// Forms
+
+iframe.form-smartsheet__support {
```

Comment:
Should be okay to remove this now, right?

Reply:
yes